magento/magento2
# Clean up lib reset

This PR aims to clean up lib css reset
#1 clean up code styles base  less magento standards
#2 remove incorrect usage of font-size-adjust: 100%; (This unneccesary)
value only accept number|none|initial|inherit. In my opinions we don't need that
#3 remove whitespace below image
remove white space below all image
Normal behavior image contain whitespace below image tag

Setting `vertical-align` removes the whitespace that appears under `img` elements when they are dropped into a page as-is. Safer alternative to using `display: block;`.

preserved previously changes, keep backward compatible

### Description (*)


### Related Pull Requests


### Fixed Issues (if relevant)

No issue available

### Manual testing scenarios (*)

Go to category or product details for make sure all images should visual correct like before the changes
icon svg, logo, images should not distorted

### Questions or comments



### Contribution checklist (*)
